.yamato/build_usd.yml

```diff
@@ -0,0 +1,65 @@
+﻿usd_versions:
+  - version: v19.11
+    dir_name: USD-19.11
+  - version: v20.08
+    dir_name: USD-20.08
```

Comment:
Do we need to build old usd versions?

Reply:
You're right we'll probably only ever go forward. I'd say we should keep the current version and the next one. So it will be 20.08 and 20.11. Or should we aim for 21.02 right away?

Reply:
I personally would just jump to 21.02 and skip 20.11, but no strong opinion